orchestration-shared-test/src/main/java/uk/gov/di/orchestration/sharedtest/extensions/AuthExternalApiStubExtension.java

```diff
@@ -56,6 +60,7 @@ public void init(Subject subjectId, Long passwordResetTime) {
                         "{"
                                 + "\"sub\": \"%s\","
                                 + "\"new_account\": true,"
+                                + "\"verified_mfa_method_type\": \"AUTH_APP\","
                                 + "\"password_reset_time\": %s"
```

Comment:
Minor refactor suggestion here, feel free to ignore, We could replace this inline JSON string constructing with something like 
```
UserInfo userInfo = new UserInfo(subjectId);
userInfo.setClaim("password_reset_time", passwordResetTime);
userInfo.setClaim("new_account", true);
userInfo.setClaim("verified_mfa_method_type", "AUTH_APP");
       
register("/userinfo", 200, "application/json", userInfo.toJSONString());

```

Just means we can get rid of a gnarly String.format

Reply:
Much nicer, done!